backend/services/implementations/classService.ts

```diff
@@ -94,6 +94,26 @@ class ClassService implements IClassService {
     return (await this.mapClassToClassDTOs(classes))[0];
   }
 
+  async getClassesByTeacherId(
+    teacherId: string,
+  ): Promise<Array<ClassResponseDTO>> {
+    let classes: Class[];
+    try {
+      classes = await MgClass.find({ teacher: { $eq: teacherId } });
+      if (!classes.length) {
+        return [];
+      }
+    } catch (error: unknown) {
```

Comment:
How would I write unit tests for error catching if there are no specific cases where errors are thrown (since if there are no classes, we return an empty array)?

Reply:
its fine lol 😀